routes/web.php

```diff
@@ -65,14 +31,8 @@
 
 
 
-
+//Likely need to remove these and throw them as the API call.
+//Right now the API call just calls the HeroController @show method, but makes more sense to just use the API call to ask for discrete data 
+//similiar to how I am getting/returning data from view Global/Hero/test.blade.php
 Route::post('/get_heroes_stats_table_data', 'HeroController@getHeroStatsTableData')->name('get_heroes_stats_table_data');
```

Comment:
//Likely need to remove these and throw them as the API call.
//Right now the API call just calls the HeroController @show method, but makes more sense to just use the API call to ask for discrete data 
//similiar to how I am getting/returning data from view Global/Hero/test.blade.php